create-regression-sprint-report/RegressionSprintReport/index.ts

```diff
@@ -0,0 +1,264 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import fetch from "node-fetch";
+import * as yaml from "js-yaml";
+
+const loginSecretParam = "ConfluenceLogin";
+const passwordSecretParam = "ConfluenceToken";
+const confluenceUrlParam = "ConfluenceUrl";
+const jiraUrlParam = "jiraUrl";
+const modulePrefix = "VirtoCommerce.";
+const platformKey = "docker.pkg.github.com/virtocommerce/vc-platform/platform";
+
+
+interface ModuleInfo
+{
+    id: string,
+    version: string,
+    sprintIssues: string,
+    oldIssues: string
+}
+
+interface ConfluencePageInfo
+{
+    type: string,
+    title: string,
+    ancestors:[
+        {
+            id:number
+        }
+    ], 
+    space:{
+        key: string
+    },
+    body:{
+        storage:{
+            value: string,
+            representation: string
+        }
+    }
+}
+
+function getHostName(url: string){
+    const regexUrl = /^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i;

```

Comment:
Need example in the comment. 
// ftp://regex-is-pain.com - invalid
// %validtUrl% - valid

Reply:
Fixed